addons/headless/functions/fnc_endMissionNoPlayers.sqf

```diff
@@ -10,19 +10,21 @@
  * None
  *
  * Example:
- * [] call acex_headless_fnc_endMissionNoPlayers;
+ * [] call acex_headless_fnc_endMissionNoPlayers
  *
  * Public: No
  */
 
-// Exit if players connected
-if !(call CBA_fnc_players isEqualTo []) exitWith {
-    GVAR(endMissionCheckDelayed) = false;
-    TRACE_1("Players are present",count (call CBA_fnc_players));
+// Exit if no players of virtual curators present
+if (call CBA_fnc_players isEqualTo [] && {allCurators isEqualTo []}) exitWith {
```

Comment:
allCurators is the logics, which will continue to exist
Checking if this connected unit is a player seems to work
`(allCurators select {isPlayer getAssignedCuratorUnit _x}) isEqualTo []`

Reply:
Without this, the test would always fail if there were zeus modules down.
Tested on dedicated with a virtual zeus and it seems to work correctly
`getAssignedCuratorUnit` returns the virtual zeus
and `isPlayer` returns true when player is connected and false on disconnect

Reply:
Thanks for the test and fix!